schemas/common/descriptors/search/searchdescriptor.description.md

```diff
@@ -0,0 +1,32 @@
+A descriptor that describes the search related annotations on the properties of a schema.  
+
+The following properties are defined as part of this descriptor: 
+
+* `search:indexable`: Boolean indicating if the property is indexable / searchable
+
+* `search:filterable`: Boolean indicating if this property can be used as a filter in search queries
+
+* `search:fetchable`: Boolean indicating if the property is returned by default in search query results
```

Comment:
Can you give an example of this `search:fetchable`, it would help me greatly. I do not find it obvious. 

PS: "fetchable" is showing as incorrect english in the chrome spell checker. It suggests "fetch-able". It does not have issues with the other -ables